.github/workflows/check-code-quality.yml

```diff
@@ -0,0 +1,21 @@
+name: check-code-quality
+on:
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  check-code-quality:
+    if: ${{ github.event.label.name == 'ready for pipeline' }}
```

Comment:
Is it possible to add some `else` part here that prints the reason why a build was skipped?
To maybe print to the developer somewhere like this:
`pipeline code-quality-check  skipped since the `'ready for pipeline'`-label is missing.